packages/web/src/scss/components/Tooltip/_TooltipPopover.scss

```
@@ -0,0 +1,96 @@
+// 1. Although it may seem pointless, having the arrow as a standalone HTML element has a purpose: the arrow can be
```

this is by default now, so we can reword this.

Something like 

```suggestion
//    The arrow must be a standalone HTML element so the Floating UI library can position it correctly.
```

---

Will be solved by @adamkudrna in his PR :)

---

Fixed.